pandas/_libs/tslib.pyx

```diff
@@ -205,7 +206,8 @@ def _test_parse_iso8601(object ts):
     elif ts == 'today':
         return Timestamp.now().normalize()
 
-    _string_to_dts(ts, &obj.dts, &out_local, &out_tzoffset)
+    if _string_to_dts(ts, &obj.dts, &out_local, &out_tzoffset) == -1:
+        raise ValueError
```

Comment:
what message is this raising?

Reply:
now I use an additional parameter `want_exc` so this construction is no longer needed